consent-ws/src/main/java/org/broadinstitute/consent/http/db/ConsentMapper.java

```diff
@@ -13,20 +14,25 @@
 
     public Consent map(int index, ResultSet r, StatementContext ctx) throws SQLException {
         Consent consent = new Consent();
-        consent.consentId = r.getString("consentId");
-        consent.requiresManualReview = r.getBoolean("requiresManualReview");
-        consent.dataUseLetter = r.getString("dataUseLetter");
-        consent.dulName = r.getString("dulName");
+        consent.setConsentId(r.getString("consentId"));
+        consent.setRequiresManualReview(r.getBoolean("requiresManualReview"));
+        consent.setDataUseLetter(r.getString("dataUseLetter"));
+        consent.setDulName(r.getString("dulName"));
         try {
-            consent.useRestriction = UseRestriction.parse(r.getString("useRestriction"));
+            consent.setUseRestriction(UseRestriction.parse(r.getString("useRestriction")));
         } catch (IOException e) {
             throw new SQLException(e);
         }
-        consent.name = r.getString("name");
+        try {
+            consent.setDataUse(DataUseDTO.parseDataUse(r.getString("dataUse")));
+        } catch (IOException e) {
+            throw new SQLException(e);
+        }
+        consent.setName(r.getString("name"));
         consent.setCreateDate(r.getTimestamp("createDate"));
         consent.setSortDate(r.getTimestamp("sortDate"));
-        consent.setLastUpdate((r.getTimestamp("lastUpdate") == null) ? null : r.getTimestamp("lastUpdate"));
-        consent.setTranslatedUseRestriction((r.getString("translatedUseRestriction") == null) ? null : r.getString("translatedUseRestriction"));
+        consent.setLastUpdate(r.getTimestamp("lastUpdate"));
+        consent.setTranslatedUseRestriction(r.getString("translatedUseRestriction"));
```

Comment:
are the null checks no longer needed?

Reply:
Technically, no, but I'm going to revert this to keep the changes down. Intellij was simplifying things for me. Reverting to reduce the risk of regression.